server/nginx-cosmosweb

```diff
@@ -0,0 +1,17 @@
+## Should be placed in /etc/nginx/site-available/ and then symlinked into /etc/nginx/sites-enabled
+## nginx config to serve the gunicorn socket to the user
+
+server {
+    listen 80;
+    server_name cosmostue.nl;
+
+    location = /favicon.ico { access_log off; log_not_found off; }
+    location /static/ {
+        root /home/prod/Cosmos;
```

Comment:
What is the reasoning for this?

Reply:
the favicon thing is because otherwise we're gonna get that in our logs all the time
and the location static, is because gunicorn can't serve the static files so we need nginx to do it